public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -216,7 +216,8 @@ const makers: SymbolMaker[] = [
   },
 ];
 
-async function prepareSVG(url: string): Promise<string> {
+async function prepareSVG(url: string, size?: number): Promise<string> {
+  const side = size ?? 100;
```

Comment:
`side` is a little bit weird for naming here IMO, maybe something like `svgSize` would be clearer?